crates/cheatcodes/spec/src/vm.rs

```diff
@@ -2404,6 +2404,10 @@ interface Vm {
     #[cheatcode(group = Crypto)]
     function rememberKey(uint256 privateKey) external returns (address keyAddr);
 
+    /// Adds private key
+    #[cheatcode(group = Crypto)]
+    function rememberKeys(string calldata mnemonic, string calldata derivationPath, uint8 count) external;
```

Comment:
```suggestion
    #[cheatcode(group = Crypto)]
    function rememberKeys(string calldata mnemonic, string calldata derivationPath, uint8 index) external;
```

Would be good to document this being a mnemonic and it probably makes sense to return `address[] keyAddr`

Reply:
+1